components/gallery/GalleryItemTabsPanel/GalleryItemOffers.vue

```diff
@@ -56,14 +56,19 @@
         <NeoSecondaryButton
           v-if="
             (props.row.caller === accountId || isOwner) &&
-            props.row.status === OfferStatusType.ACTIVE
+            props.row.status === OfferStatusType.ACTIVE &&
+            expirationTime(props.row.expiration) !== 'Expired'
```

Comment:
pls make a getter function for the condition `props.row.status === OfferStatusType.ACTIVE &&
            expirationTime(props.row.expiration) !== 'Expired'` to avoid copy paste